### ISSUE TITLE
layouts in gule 

### ISSUE THREAD

**Issue by crypt17** [engstrand-config/dwl-guile#43]
I was wondering if you can considered allowing for the setup of layouts from guile code.

I feel that this feature would allow many things that are now patches to the C code base to be easily implemented

**Comment by Frewacom** [comment]
Agreed, it would be nice. However, there is quite a lot of work needed to allow for such scripting capabilities.

**Comment by crypt17** [comment]
I was thinking of something like a function layout_custom(monitor *m, char *option)

This would just take a monitor and a selector.

It would then compute how many windows needed to be laid out and pass that through to a central guile procedure which would return a list of dimensions so that the wndows could then be laid out.

I do not know the code base well enough yet to know if this would work.
